Rename negative_ttl into default_ttl (#39)
systemli/ansible-role-bind9: README.md

- { name: master_name, addresses: [master_ip] }
bind9_recursor: our_network
```
### Dynamic zones
So called "dynamic" zones' records are defined through YAML ansible variable `bind9_zones_dynamic` which is parsed by [`bind/zones/db.template.j2`](templates/bind/zones/db.template.j2) template.

> As there can be several zones, and zones definitions can be long, zones vars are worthly defined in a different vars' file, for instance `host_vars/master_name/vars/YY_zones.yml`, and  `bind9_zones_dynamic` can be splited in several variables, that can bie defined in specific files. In `YY_zones.yml` we may have:

Reviewer: ```suggestion
As there can be several zones, and zone definitions can be long, zone vars are worthily defined in a different vars file, for instance `host_vars/master_name/vars/YY_zones.yml`.  `bind9_zones_dynamic` can be split in several variables, which can be defined in specific files, as in the example below.

In `YY_zones.yml` we may have:
```

Author: > Thank you for your PR. 
Thanks for your review, @t2d !

> The addition to the README will make it much more easy to understand. 
Great!

> I'm not sure the renaming of `negative_ttl` to `default_ttl` is really necessary. Negative TTL seems to be the technically correct term.

Sorry, I insist. Look at [this paragraph in Bind9 ARM](https://bind9.readthedocs.io/en/latest/chapter3.html#setting-ttls). The right meaning of this parameter is `default_ttl`, not `negative_ttl`.  The negative TTL of a zone is defined by the last parameter of the SOA resource record, the often called the _minimum_ field.  The variable of our role we are trying to name defines the `$TTL` BIND macro of a zone's file,  which sets the default *positive* TTL of resource records (RRs),  used if TTL is not explicitly set as the second field of the RR line.

Reviewer: Okay, learned something today. Thanks.
Are you accepting my spelling adjustments?

Author: > Okay, learned something today. Thanks.

Great. Only stupid people don't change their mind :-)

> Are you accepting my spelling adjustments?

Yes, thank you! Strictly, I think "vars' file" is grammatically correct ([as in "my parents' friends" here](https://www.oxfordlearnersdictionaries.com/us/grammar/online-grammar/possessive-s-and-s)), but "vars file" is commonly accepted and perfectly understandable.
